packages/react-composites/tests/browser/chat/fake-adapter/FileSharing.test.ts

```diff
@@ -46,6 +46,8 @@ test.describe('Filesharing SendBox', async () => {
         ]
       })
     );
+    // timeout for mobile landscape needed because scrollbar appears momentarily when new message in thread causes screen overflow
```

Comment:
why does it only show on mobile landscape? Is there a height bug somewhere we need to fix?

Reply:
I've only seen tests fail when there is a message with a filecard, does it show on all messages even ones with just chats?

Reply:
Oh I guess because only mobile landscape has a full screen of messages hmmm

Reply:
I am quite sure it only shows up for mobile landscape but I'm not sure why. Here is the video of the test in debug mode:
https://user-images.githubusercontent.com/79475487/172831382-bf36a330-b6d8-4d34-9f4d-e206d22f611b.mp4
I notice the config difference between Mobile Android Portrait and Mobile Android Landscape is more than just screen size.
```
    {
      name: 'Mobile Android Portrait',
      use: {
        /*
          userAgent: 'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4641.0 Mobile Safari/537.36',
          viewport: { width: 360, height: 640 },
          deviceScaleFactor: 3,
          isMobile: true,
          hasTouch: true,
          defaultBrowserType: 'chromium'
        */
        ...devices['Nexus 5'],
        launchOptions: { ...chromeLaunchOptions }
      }
    },
    {
      name: 'Mobile Android Landscape',
      use: {
        // Nexus 5 user agent string, taken from node_modules/.../playwright-core/.../deviceDescriptorsSource.json
        userAgent:
          'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4641.0 Mobile Safari/537.36',
        // Support smallest supported mobile viewport (iPhone 5/SE) ({ width: 568, height: 320 })
        viewport: { width: 568, height: 320 },
        deviceScaleFactor: 2,
        isMobile: true,
        hasTouch: true,
        defaultBrowserType: 'chromium',
        launchOptions: { ...chromeLaunchOptions }
      }
    }
```
Will try other configs and see if I can make the scrollbar disappear. Also I'll check if the momentary scrollbar happens in for Mobile Android Portrait

Reply:
The scrollbar should only be the message thread, and there should be more of a gap at the bottom:
![image](https://user-images.githubusercontent.com/2684369/172951442-33115ef9-8b30-44bf-b160-f04ef0b2c199.png)

Whereas in the bug filed for this it looks like its a page scrollbar showing:
![image](https://user-images.githubusercontent.com/2684369/172951604-e8597793-9248-4f4b-a205-c4a1482139d2.png)

Which makes me think this is an issue with file sharing specifically

Reply:
Thinking more, maybe not filesharing specifically, but perhaps the way the ChatComposite is sized in the ui test app, it should never be greater than the page size

Reply:
Either way a fix for this should be its own PR as fixing this seems unrelated to custom data model tests (unless we see new tests failing other than the one filed in this bug [Bug 2883029](https://skype.visualstudio.com/SPOOL/_workitems/edit/2883029): [Web] Flakey filesharing UI test due to scrollbar)

Reply:
looking at the config above, why does the landscape vs portrait have different aspect ratios? shouldn't the landscape dimensions be: `{width: 640, height: 360}`?

Reply:
I think James' inkling about this being a FileSharing issue is right. But also mobile scrollbars are transient.
Test on Desktop with overflowing file sharing messages:
https://user-images.githubusercontent.com/79475487/172978136-284fd954-2699-495e-abe5-58035618881c.mp4
Test on Mobile portrait with overflowing file sharing messages:
https://user-images.githubusercontent.com/79475487/172978402-0918ca6d-e369-46de-af66-e6111a4deb2c.mp4
Test on Mobile landscape with overflowing  text messages:
https://user-images.githubusercontent.com/79475487/172978522-97bb6fd8-c103-4a8b-a777-895ebba32343.mp4